### PR TITLE
Update dashboard-statement-details.md

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/13087
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dashboard/dashboard-statement-details.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ Currently, the fast plan binding feature does not support the following types of
 - Queries that access TiFlash
 - Queries that join three or more tables
 
+The current feature requires users to have SUPER privilege to use. If you encounter permission issues while using it, please refer to [TiDB Dashboard User Management](dashboard-user.md) to add the necessary privileges.
```
Comment (ran-huang) [inline dashboard/dashboard-statement-details.md:60]:
```suggestion
To use this feature, you must have the `SUPER` privilege. If you encounter permission issues while using it, refer to [TiDB Dashboard User Management](dashboard-user.md) to add the necessary privileges.
```

File: dashboard/dashboard-user.md [hunk 2]
```diff
@@ -53,6 +58,10 @@ If an SQL user does not meet the preceding privilege requirements, the user fail
 
     -- To modify the configuration items on the interface after signing in to TiDB Dashboard, the user-defined SQL user must be granted with the following privilege.
     GRANT SYSTEM_VARIABLES_ADMIN ON *.* TO 'dashboardAdmin'@'%';
+    
+    -- To use the [Quick Bind Executions Plan](dashboard-statement-details.md) feature on the interface after signing in to TiDB Dashboard, the user-defined SQL user must be granted with the following privilege.
```
Comment (ran-huang) [inline dashboard/dashboard-user.md:62]:
```suggestion
    -- To use the Fast Bind Executions Plan feature (https://docs.pingcap.com/tidb/v6.6/dashboard-statement-details#fast-plan-binding) on the interface after signing in to TiDB Dashboard, the user-defined SQL user must be granted with the following privilege.
```

File: dashboard/dashboard-user.md [hunk 3]
```diff
@@ -33,6 +33,11 @@ For details about how to control and manage TiDB SQL users, see [TiDB User Accou
 
     - SYSTEM_VARIABLES_ADMIN
 
+- To use the [Quick Bind Executions Plan](dashboard-statement-details.md) feature on the interface after signing in to TiDB Dashboard, the SQL user must also have the following privilege:
```
Comment (ran-huang) [inline dashboard/dashboard-user.md:36]: 文档里现在用的是 fast，统一成 fast 还是 quick 吧

```suggestion
- To use the [Fast Bind Executions Plan](dashboard-statement-details.md#fast-plan-binding) feature on the interface after signing in to TiDB Dashboard, the SQL user must also have the following privilege:
```

File: dashboard/dashboard-user.md [hunk 4]
```diff
@@ -68,20 +77,25 @@ If an SQL user does not meet the preceding privilege requirements, the user fail
 
     -- To modify the configuration items on the interface after signing in to TiDB Dashboard, the user-defined SQL user must be granted with the following privilege.
     GRANT SYSTEM_VARIABLES_ADMIN ON *.* TO 'dashboardAdmin'@'%';
+    
+    -- To use the [Quick Bind Executions Plan](dashboard-statement-details.md) feature on the interface after signing in to TiDB Dashboard, the user-defined SQL user must be granted with the following privilege.
```
Comment (ran-huang) [inline dashboard/dashboard-user.md:81]: ditto

Comment (ran-huang) [inline dashboard/dashboard-user.md:81]:
```suggestion
    -- To use the Fast Bind Executions Plan feature (https://docs.pingcap.com/tidb/v6.6/dashboard-statement-details#fast-plan-binding) on the interface after signing in to TiDB Dashboard, the user-defined SQL user must be granted with the following privilege.
```

**Review by ran-huang — APPROVED** [review]

File: dashboard/dashboard-user.md [hunk 5]
```diff
@@ -53,6 +58,10 @@ If an SQL user does not meet the preceding privilege requirements, the user fail
 
     -- To modify the configuration items on the interface after signing in to TiDB Dashboard, the user-defined SQL user must be granted with the following privilege.
     GRANT SYSTEM_VARIABLES_ADMIN ON *.* TO 'dashboardAdmin'@'%';
+    
+    -- To use the Fast Bind Executions Plan feature (https://docs.pingcap.com/tidb/v6.6/dashboard-statement-details#fast-plan-binding) on the interface after signing in to TiDB Dashboard, the user-defined SQL user must be granted with the following privilege.
```
Comment (ran-huang) [inline dashboard/dashboard-user.md:62]:
```suggestion
    -- To use the Fast Bind Executions Plan feature (https://docs.pingcap.com/tidb/dev/dashboard-statement-details#fast-plan-binding) on the interface after signing in to TiDB Dashboard, the user-defined SQL user must be granted with the following privilege.
```

File: dashboard/dashboard-user.md [hunk 6]
```diff
@@ -68,20 +77,25 @@ If an SQL user does not meet the preceding privilege requirements, the user fail
...
```
Comment (ran-huang) [inline dashboard/dashboard-user.md:81]:
```suggestion
    -- To use the Fast Bind Executions Plan feature (https://docs.pingcap.com/tidb/dev/dashboard-statement-details#fast-plan-binding) on the interface after signing in to TiDB Dashboard, the user-defined SQL user must be granted with the following privilege.
```

File: dashboard/dashboard-statement-details.md [hunk 7]
```diff
@@ -57,6 +57,8 @@ Currently, the fast plan binding feature does not support the following types of
 - Queries that access TiFlash
 - Queries that join three or more tables
 
+To use this feature, you must have the SUPER privilege. If you encounter permission issues while using it, refer to [TiDB Dashboard User Management](dashboard-user.md) to add the necessary privileges.
```
Comment (ran-huang) [inline dashboard/dashboard-statement-details.md:60]:
```suggestion
To use this feature, you must have the SUPER privilege. If you encounter permission issues while using it, refer to [TiDB Dashboard User Management](/dashboard/dashboard-user.md) to add the necessary privileges.
```

File: dashboard/dashboard-user.md [hunk 8]
```diff
@@ -33,6 +33,11 @@ For details about how to control and manage TiDB SQL users, see [TiDB User Accou
 
     - SYSTEM_VARIABLES_ADMIN
 
+- To use the [Fast Bind Executions Plan](dashboard-statement-details.md#fast-plan-binding) feature on the interface after signing in to TiDB Dashboard, the SQL user must also have the following privilege:
```
Comment (ran-huang) [inline dashboard/dashboard-user.md:36]:
```suggestion
- To use the [Fast Bind Executions Plan](/dashboard/dashboard-statement-details.md#fast-plan-binding) feature on the interface after signing in to TiDB Dashboard, the SQL user must also have the following privilege:
```

File: dashboard/dashboard-user.md [hunk 9]
```diff
@@ -33,6 +33,11 @@ For details about how to control and manage TiDB SQL users, see [TiDB User Accou
 
     - SYSTEM_VARIABLES_ADMIN
 
+- To use the [Fast Bind Executions Plan](/dashboard/dashboard-statement-details.md#fast-plan-binding) feature on the interface after signing in to TiDB Dashboard, the SQL user must also have the following privilege:
```
Comment (TomShawn) [inline dashboard/dashboard-user.md:36]:
```suggestion
- To use the [Fast Bind Executions Plan](/dashboard/dashboard-statement-details.md#fast-plan-binding) feature on the interface after signing in to TiDB Dashboard, the SQL user must also have the following privileges:
```

File: dashboard/dashboard-user.md [hunk 10]
```diff
@@ -53,6 +58,10 @@ If an SQL user does not meet the preceding privilege requirements, the user fail
 
     -- To modify the configuration items on the interface after signing in to TiDB Dashboard, the user-defined SQL user must be granted with the following privilege.
     GRANT SYSTEM_VARIABLES_ADMIN ON *.* TO 'dashboardAdmin'@'%';
+    
+    -- To use the Fast Bind Executions Plan feature (https://docs.pingcap.com/tidb/dev/dashboard-statement-details#fast-plan-binding) on the interface after signing in to TiDB Dashboard, the user-defined SQL user must be granted with the following privilege.
```
Comment (TomShawn) [inline dashboard/dashboard-user.md:62]:
```suggestion
    -- To use the Fast Bind Executions Plan feature (https://docs.pingcap.com/tidb/dev/dashboard-statement-details#fast-plan-binding) on the interface after signing in to TiDB Dashboard, the user-defined SQL user must be granted with the following privileges.
```

File: dashboard/dashboard-user.md [hunk 11]
```diff
@@ -68,20 +77,25 @@ If an SQL user does not meet the preceding privilege requirements, the user fail
...
```
Comment (TomShawn) [inline dashboard/dashboard-user.md:81]:
```suggestion
    -- To use the Fast Bind Executions Plan feature (https://docs.pingcap.com/tidb/dev/dashboard-statement-details#fast-plan-binding) on the interface after signing in to TiDB Dashboard, the user-defined SQL user must be granted with the following privileges.
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

**Pull request merged**